.circleci/config.yml

```diff
@@ -635,6 +635,11 @@ workflows:
           container: "nsc"
           requires:
             - sanity-check
+      - build-container:
+          name: "build-monitoring-nsc"
+          container: "monitoring-nsc"
+          requires:
+            - sanity-check
```

Comment:
You probably want to update `docker-push-latest` too.

Reply:
You are right, fixed this : c780836